src/app/lib/utilities/preprocessor/rules/cpsAssetPage/addHeadlineBlock/models/index.js

```diff
@@ -0,0 +1,89 @@
+export const getHeadlineBlock = headlineText => ({
```

Comment:
These could be produced by a helper, just thought the copy and paste seemed more explicit, happy to change if reviewers think so